packages/bsky/src/hydration/hydrator.ts

```diff
@@ -737,6 +752,33 @@ export class Hydrator {
     return { follows, followBlocks }
   }
 
+  async hydrationBidirectionalBlocks(
+    didMap: Map<string, string[]>,
+  ): Promise<BidirectionalBlocks> {
+    const pairs: RelationshipPair[] = []
+
+    for (const [uri, followers] of didMap) {
+      for (const follower of followers) {
+        pairs.push([didFromUri(uri), follower])
+      }
+    }
```

Comment:
It looks like `didMap` is `DID -> DID[]`, in which case we don't have a URI to `didFromUri()` with.